modular_skyrat/modules/colony_fabricator/code/machines/solar_panels.dm

```diff
@@ -0,0 +1,61 @@
+// Solar panels
+
+/obj/machinery/power/solar/quickdeploy
```

Comment:
Snake_case means that there's an underscore between words, otherwise it's just lowercase and that's probably one of the worst ways of writing variable names.
```suggestion
/obj/machinery/power/solar/quick_deploy
```